### PR TITLE
Some commits from master-wowl that are ready

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

File: addons/web/static/src/search/control_panel/control_panel.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        const display = Object.assign(
+            {
+                "top-left": true,
+                "top-right": true,
+                "bottom-left": true,
+                "bottom-right": true,
+            },
+            this.props.display
+        );
+        display.top = display["top-left"] || display["top-right"];
```
Comment (Polymorphe57) [inline addons/web/static/src/search/control_panel/control_panel.js:25]: logic should be improved

File: addons/web/static/src/core/dropdown/dropdown.js [hunk 2]
```diff
@@ -48,12 +48,6 @@ export class Dropdown extends Component {
```
Comment (Polymorphe57) [inline addons/web/static/src/core/dropdown/dropdown.js:48]: remove it from PR

**Comment by kebeclibre** [comment]
robodoo r+

**Comment by kebeclibre** [comment]
robodoo rebase-ff

**Pull request closed without merging**